pset-iterators.js

```diff
@@ -137,7 +159,11 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 5:')
 
 // Add your code below this line
+const livingPresidents = presidents.filter(function(president){
+  return president.left_office === 'null'
+})
 
+console.log(livingPresidents)
```

Comment:
@ChicoMangusta  Good job on this solution here using the filter method to return all of the presidents who are currently living.

Take a look at this approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L148)